### PR TITLE
Unittesting

### PR DESCRIPTION
**Pull request by KizeKaze**
added unittesting and tested every function known to man

### CODE REVIEW

**Comment by KizeKaze** [comment]
fixes #68

**Review by tony1542 — CHANGES_REQUESTED** [review]
There are hard-coded values in here that aren't in the `seed.sql` script so it isn't  guaranteed they will exist by the time the tests are ran. could you update the `seed.sql` with some dummy data and refer to those records in the unit tests?

File: database_seed.sql [hunk 1]
```diff
@@ -12,6 +12,21 @@ CREATE TABLE item
...
+INSERT INTO fast_food.item (name, description, cost, type_id) VALUES ('Carrots', '1 Baby carrots in a bag', 1.99, 2);
+INSERT INTO fast_food.item (name, description, cost, type_id) VALUES ('Apple', 'An Apple', 0.50, 1);
+INSERT INTO fast_food.item (name, description, cost, type_id) VALUES ('WaterMelon', 'A juicy watermelon, only the best', 2.59, 1);
+INSERT INTO fast_food.item (name, description, cost, type_id) VALUES ('Coke 12 pack', '12 pack of tasty coke', 5.99, 8);
+INSERT INTO fast_food.item (name, description, cost, type_id) VALUES ('Mt Dew 12 pack', '12 pack of Mt Dew', 5.99, 8);
+INSERT INTO fast_food.item (name, description, cost, type_id) VALUES ('Eggs', 'a dozen eggs', 0.99, 7);
+INSERT INTO fast_food.item (name, description, cost, type_id) VALUES ('Chicken Breast', '4 juicy chicken breast', 7.99, 3);
+INSERT INTO fast_food.item (name, description, cost, type_id) VALUES ('ground beef', '1 pound of ground beef', 4.99, 4);
+INSERT INTO fast_food.item (name, description, cost, type_id) VALUES ('Crab legs', 'Crab legs of the sea', 3.99, 5);
+INSERT INTO fast_food.item (name, description, cost, type_id) VALUES ('Coffee', '942 MG of caffine', 0.69, 9);
```
Comment (tony1542) [inline database_seed.sql]: lol

File: tests/MenuTest.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+        $item = self::$Menu->getItem($item_id);
+        $this->assertNotEmpty($item);
+        $this->assertIsArray($item);
+    }
+
+    public function testUpdateItem()
+    {
+
+        $sql = 'SELECT MAX(id) AS item_id FROM item';
```
Comment (tony1542) [inline tests/MenuTest.php]: This starting query isn't helpful for the rest of the this unit test and can be stripped away

File: tests/MenuTest.php [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        self::$id[] = $inserted_id;
+
+        $name = 'wowie';
+        $description = 'Sometimes';
+        $cost = '17.99';
+        $type_id = '13';
+
+        self::$Menu->updateItem($inserted_id, $name, $description, $cost, $type_id);
+
+        $sql = "SELECT name FROM item WHERE id = $inserted_id";
```
Comment (tony1542) [inline tests/MenuTest.php]: You should be grabbing each of the 4 columns that were updated in the `updateItem()` call and asserting them as well here

**Review by tony1542 — CHANGES_REQUESTED** [review]

**Comment by KizeKaze** [comment]
forgot 4 functions in menu.php

**Review by tony1542 — APPROVED** [review]
Had to run `$ "vendor/bin/phpunit" tests --bootstrap ./vendor/autoload.php`, got 100% passing. good job

**Pull request merged**